src/utils/validation/postSchema.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from 'zod';
+
+const postSchema = z.object({
+  postContent: z
+    .string()
+    .nonempty('Post cannot be empty')
+    .min(2, 'Post must be at least 2 characters long'),
+  postImage: z
+    .string()
+    .refine((value) => /\.(jpe?g|png|gif)$/gi.test(value), {
+      message: 'Invalid image format. Only JPEG, PNG, and GIF are supported.',
+    })
+    .optional()
+    .nullable(),
+});

```

Comment:
I think the validation doesn't work.
Coz I try to upload `svg` file and it was created